airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -652,6 +656,11 @@ async def test_discover(self, connector_config, docker_runner: ConnectorRunner):
         assert len(catalog_messages) == 1, "Catalog message should be emitted exactly once"
         assert catalog_messages[0].catalog, "Message should have catalog"
         assert catalog_messages[0].catalog.streams, "Catalog should contain streams"
+        assert self.streams_are_uniquely_named(catalog_messages[0].catalog.streams), "Catalog should have uniquely named streams"
```

Comment:
Can we update this to output which streams are duplicates?
```suggestion
        duplicate_streams = get_duplicates(catalog_messages[0].catalog.streams)
        assert len(duplicate_streams) == 0, f"Catalog should have uniquely named streams. Duplicate stream(s) found: {duplicate_streams}"
        
def get_duplicates(strings):
    counts = {string: strings.count(string) for string in strings}
    duplicates = [string for string, count in counts.items() if count > 1]
    return duplicates
```